pyproject.toml

```diff
@@ -50,6 +50,13 @@ override-dependencies = [
     "torchaudio; sys_platform == 'never'",
     "torchvision; sys_platform == 'never'",
     "triton; sys_platform == 'never'",
+    
+    # Skip packages only on s390x, expected to be pre-installed
+    "vllm ; platform_machine != 's390x'",
+    "ray; platform_machine != 's390x'",
+    "ray[default]; platform_machine != 's390x'",
```

Comment:
I think this is causing all the `ray[default]` deps to be pulled in for x86 (see all the uv.lock additions) which is not what we want